CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## 3.3.0 (under development)
+## 3.3.0 (2019-03-27)
+
+- Upgrade job-dsl-core to 1.72.
```

Comment:
What about the other dependency upgrades? Even if they may not be user-facing, should we have something generic like "Upgrade several implementation dependencies" here?

Reply:
I thought about it but decided against it because such upgrades also weren't mentioned in previous changelog entries.